backend/docker/sql/views/signed_extrinsics_per_day.sql

```diff
@@ -0,0 +1,11 @@
+CREATE VIEW signed_extrinsics_per_day AS
```

Comment:
Add `view` postfix. `signed_extrinsics_per_day_view`

Reply:
added

Reply:
Add to the view name as well. Not only file.